about.html

```diff
@@ -91,25 +91,31 @@
                         </a>
                       </div>
                       <!-- IMAGE SEARCH -->
-                      <div class="image search">
-                        <div data-cmp-lazythreshold="0" data-cmp-src="/content/leaderimpact/lang/en2/_jcr_content/content-parsys/container/container_2037006891/container/image.coreimg{.width}.svg/1669058888816/search.svg" data-asset="/content/dam/LeaderImpact/site-./assets/search.svg" data-title="search" id="image-fd5f155c3f" class="cmp-image" itemscope="" itemtype="http://schema.org/ImageObject">
-                          <img src="./assets/search.svg" class="cmp-image__image" itemprop="contentUrl" data-cmp-hook-image="image" alt="search">
-                        </div>
+                      <div class="button image search">
+                        <button type="button" id="button-5487aab983" class="cmp-button" data-cmp-clickable="" data-cmp-data-layer="{&quot;button-5487aab983&quot;:{&quot;@type&quot;:&quot;core-components-examples/components/button&quot;,&quot;dc:title&quot;:&quot;Button&quot;}}">
+                          <span class="cmp-button__icon cmp-button__icon--search search" aria-hidden="true">
+                            <img src="./assets/search.svg" class="cmp-image-button__icon" itemprop="contentUrl" data-cmp-hook-image="image" alt="Close">
```

Comment:
@Prystal254 Button components don't use `<img />` for the icons. You need to use a pseudo element.